rollup.mathjax.ts

```diff
@@ -19,4 +19,4 @@ export default {
     }),
     commonjs(),
   ],
-};
```

Comment:
Try to avoid these small unrelated changes. This might be caused by a difference between windows and mac/linux.